modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -0,0 +1,86 @@
+:_content-type: PROCEDURE
+:description: Starting a new workspace with a clone of a Git repository
+:keywords: start-new-workspace, start-a-new-workspace, how-to-start-new-workspace, how-to-start-a-new-workspace, starting-a-new-workspace, clone-git-repository, clone-a-git-repository, how-to-start-workspace, how-to-start-a-workspace
+:navtitle: Starting a new workspace with a clone of a Git repository
+// :page-aliases:
+
+[id="starting-a-new-workspace-with-a-clone-of-a-git-repository_{context}"]
+= Starting a new workspace with a clone of a Git repository
+
+Working with {prod-short} in your browser involves multiple URLs:
+
+* URL of your organization's {prod-short} instance (as part of all the following URLs)
+* URL of the *Workspaces* page of your {prod-short} dashboard (workspace control panel)
+* [.underline]#URLs for starting a new workspace#
+* URLs of your workspaces in use
+
+With {prod-short}, you can visit a URL in your browser to start a new workspace that contains a clone of a Git repository. So you can have your remote origin or fork cloned into a new starting workspace. Or you can have a common repository of a project cloned into a new starting workspace. With such Git providers as GitHub and GitLab, you can also specify the branch for the new starting workspace to checkout.
+
+NOTE: The *Create Workspace* page of your {prod-short} dashboard has the *Git Repo URL ** field as the limited-functionality GUI alternative for entering the URL of a Git repository to start a new workspace.
+
+.Prerequisites
+
+* Your organization has a running instance of {prod-short}.
+* The administrator of your organization's {prod-short} instance or the repository maintainer or the README author has provided you with the URL of your organization's {prod-short} instance that you need as a user to access your {prod-short} dashboard pages and start and manage your workspaces. This URL is a fully qualified domain name (FQDN): `{prod-url}`.
+* Your Git repository maintainer keeps the `devfile.yaml` or `.devfile.yaml` file in the root directory of the Git repository. (For alternative file names and file paths, see xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[].)
++
+IMPORTANT: Although the devfile enables full functionality of a workspace, you can also start a new workspace by supplying the URL of a Git repository without a devfile. This results in a new workspace with a clone of the Git repository and with the defaults such as the Che-Theia IDE and the Universal Developer Image.
+//provide xrefs to a concept module about the Universal Developer Image and, if applicable, devfile-less defaults for new workspaces. max-cx
++
+TIP: Advanced users can compose and use their own devfiles.
```

Comment:
You might want to avoid "stacking" admonitions. Also, this one looks out of place here in the Prerequisites section.

Reply:
Yep, I completely agree with you. I haven't liked this admonition stack from the beginning.
I'll try to rethink it.